src/api/transformers/ContentTransformer.ts

```diff
@@ -211,6 +210,9 @@ const contentData = {
     stripeCountry: ["readonly", () => config.stripe.country],
     taxRateEnabled: ["option", "tax-rate-enabled", "bool"],
     taxRate: ["option", "tax-rate-percentage", "int"]
+  }),
+  telegram: withValue<"telegram">({
+    welcomeMessageMd: ["option", "telegram.welcome-message.md", "text"]
```

Comment:
I think this one should be a `"data"` type value rather than `"option"`. The difference is basically whether it uses the `OptionsService` or the `Content` table (`"content"` would probably have been a better name than `"data"`!).

The `OptionsService` is a in-memory store with database persistence, it should really only be used for settings where they need to be read synchronously elsewhere in the application. In this case this value is only ever passed back from the content API so doesn't need to be an option.

See the definition of `profile.introMessage` above for an example.